pkg/apis/core/v1/conversion.go

```diff
@@ -382,6 +385,10 @@ func Convert_core_Pod_To_v1_Pod(in *core.Pod, out *v1.Pod, s conversion.Scope) e
 	// remove this once the oldest supported kubelet no longer honors the annotations over the field.
 	out.Annotations = dropInitContainerAnnotations(out.Annotations)
 
+	if err := HandleSeccompVersionSkew(in, out); err != nil {
+		return errors.Wrap(err, "handling seccomp version skew")
+	}
```

Comment:
rather than doing this in conversion, I expected it to be contained in pkg/registry/core/pod/strategy.go and called from `podStrategy#PrepareForCreate`

Reply:
That’s a great hint and saves me time, thank you!